src/data/data.js

```diff
@@ -109,7 +109,7 @@ export default [
                 id: 7,
                 name: 'BAZAR RÓŻYCKIEGO',
                 adres: 'Lokalizacja: ul. Targowa 54',
-                haslo: 'Legenda praskiej ulicy',
+                haslo: 'LEGENDA PRASKIEJ ULICY',
```

Comment:
hmm, I believe that you want to display UPPPER CASE catchphase (btw catchphase or catchword not "haslo" 😊 ) - you can do it when displaying it, no need to change data.

Please just use `toUpperCase` (while displaying) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase

or do it by CSS https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform